src/localization/tableStrings.js

```diff
@@ -55,56 +55,60 @@ export const tableStrings = new LocalizedStrings({
     batches: 'BATCHES',
     batch_name: 'BATCH NAME',
     expiry: 'EXPIRY',
+    price: 'PRICE',
+    unit: 'UNIT',
   },
   fr: {
-    actual_quantity: 'Quantité\nRéelle',
-    available_stock: 'Quantité\ndisponible',
-    batch_expiry: 'Expiration',
-    batch: 'Lot',
-    category: 'Catégorie',
+    actual_quantity: 'QUANTITÉ\nRÉELLE',
+    available_stock: 'QUANTITÉ\nDISPONIBLE',
+    batch_expiry: 'EXPIRATION',
+    batch: 'LOT',
+    category: 'catégorie',
```

Comment:
this feel nit-picky.. can it be an uppercase 'C' please?

Reply:
058f7a33b0b60f53970cabd64e80354d79541574